CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+0.8.3 / 2024-07-18
+===================
+## Kover Aggregation Plugin
+* Added support for Android projects in Kover Aggregation Plugin
+
+## Kover Maven Plugin
+* [`#51`](https://github.com/Kotlin/kotlinx-kover/issues/51) Implemented Kover Maven Plugin
```

Comment:
Sounds major to me :)

Reply:
Yeah, you can write a larger announcement about it